renderdoc/os/posix/apple/apple_helpers.mm

```diff
@@ -56,3 +58,13 @@ bool apple_IsKeyPressed(int appleKeyCode)
 {
   return s_keysPressed[appleKeyCode];
 }
+
+rdcstr apple_GetExecutablePathFromAppBundle(const char *appBundlePath)
+{
+  NSString *path = [NSString stringWithCString:appBundlePath encoding:NSASCIIStringEncoding];
```

Comment:
Should we be using ASCII string encoding here? rdcstr is considered UTF-8 in all cases. Will this work if the path uses unicode characters?

Reply:
switched to UTF8 encoding